File: AppGenerateFiles/dbo/Model/Catalogo_Nacionalidad.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Nacionalidad : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_nacionalidad { get; set; }
       public string? nombre { get; set; }
       public string? nacionalidad { get; set; }
       public int? ponderacion { get; set; }
       public int? puntaje { get; set; }
       public string? clasificacion { get; set; }
       [OneToMany(TableName = "Catalogo_Departamento", KeyColumn = "id_nacionalidad", ForeignKeyColumn = "id_nacionalidad")]
       public List<Catalogo_Departamento>? Catalogo_Departamento { get; set; }
       [OneToMany(TableName = "Catalogo_Inversores", KeyColumn = "id_nacionalidad", ForeignKeyColumn = "id_nacionalidad")]
       public List<Catalogo_Inversores>? Catalogo_Inversores { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Nacionalidad.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Departamento }  from './Catalogo_Departamento.js'
import { Catalogo_Inversores }  from './Catalogo_Inversores.js'
class Catalogo_Nacionalidad extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_nacionalidad;
   /**@type {String}*/ nombre;
   /**@type {String}*/ nacionalidad;
   /**@type {Number}*/ ponderacion;
   /**@type {Number}*/ puntaje;
   /**@type {String}*/ clasificacion;
   /**@type {Array<Catalogo_Departamento>} OneToMany*/ Catalogo_Departamento;
   /**@type {Array<Catalogo_Inversores>} OneToMany*/ Catalogo_Inversores;
}
export { Catalogo_Nacionalidad }
